modules/porous_flow/include/materials/PorousFlowFluidStateFlashBase.h

```diff
@@ -53,14 +53,36 @@ class PorousFlowFluidStateFlashBase : public PorousFlowVariableBase
   virtual void initQpStatefulProperties() override;
   virtual void computeQpProperties() override;
 
+  /// Data structure to pass calculated thermophysical properties
+  struct FluidStateProperties
```

Comment:
I like this idea and the resulting code simplifications. I'll have a crack at this today - thanks @rwcarlsen